### PR TITLE
webreg: fix job page

### PR DESCRIPTION
**Pull request by AlexNPavel**
Fixes `/job` URLs that broke due to implementation of metadata in the config agent.

### CODE REVIEW

**Comment by AlexNPavel** [comment]
Example job on server running this PR: https://server-ci-tools-773.svc.ci.openshift.org/job?org=openshift&repo=aws-pod-identity-webhook&branch=master&test=e2e-aws

**Comment by droslean** [comment]
/hold

If this fixes the breaking changes why it is still failing?

File: pkg/webreg/webreg.go [hunk 1]
```diff
@@ -1687,31 +1687,28 @@ func WebRegHandler(regAgent agents.RegistryAgent, confAgent agents.ConfigAgent)
...
+				referenceHandler(regAgent, w, req)
 				return
-			} else if splitURI[0] == "job" {
-				jobHandler(regAgent, confAgent, w, req)
+			}
+			if _, ok := chains[splitURI[1]]; ok {
+				chainHandler(regAgent, w, req)
+				return
+			}
+			if _, ok := workflows[splitURI[1]]; ok {
```
Comment (droslean) [inline pkg/webreg/webreg.go:1706]: nit: assign the `splitURI[1]` to a var to avoid multiple lookups.

File: pkg/webreg/webreg.go [hunk 2]
```diff
@@ -1687,31 +1687,28 @@ func WebRegHandler(regAgent agents.RegistryAgent, confAgent agents.ConfigAgent)
...
-				if _, ok := chains[splitURI[1]]; ok {
-					chainHandler(regAgent, w, req)
-					return
-				}
-				if _, ok := workflows[splitURI[1]]; ok {
-					workflowHandler(regAgent, w, req)
-					return
-				}
-				writeErrorPage(w, fmt.Errorf("Registry element %s not found", splitURI[1]), http.StatusNotFound)
+		} else if len(splitURI) == 2 && splitURI[0] == "registry" {
```
Comment (droslean) [inline pkg/webreg/webreg.go:1696]: nit: I would really like to have this `registry` value to a constant with a name related to its usage.

**Comment by bbguimaraes** [comment]
https://github.com/openshift/release/pull/8943

/retest

**Comment by stevekuznetsov** [comment]
/lgtm
/hold cancel

The deployment seems to work fine

**Pull request merged**